pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation.adoc

```diff
@@ -0,0 +1,41 @@
+[[apim-kubernetes-operator-installation]]
+= GKO installation and deployment
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_installation.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+You can deploy the Gravitee Kubernetes Operator [GKO] in two ways:
+
+1. On a production Kubernetes cluster.
+2. On a local Kubernetes cluster (for testing purposes only).
```

Comment:
I don't know if we should talk about k3d here. By the end of the day, deploying the operator should be the same regardless of the environment. 

k3d is mainly a tool we use for development purpose that we share as a facility to make users able to get a local environment promptly and play with the product.

I think k3d should be mentioned as an option in the quick start guide, but that's all.

Something like, "If you don't have a local cluster running APIM on hand, you can use this command to set it up

```shell
curl -s https://raw.githubusercontent.com/gravitee-io/gravitee-kubernetes-operator/chore-inline-k3d-script/scripts/k3d.sh| bash
```

BTW I don't remember if we tested it on a Linux OS so maybe we should mention that the script is intended to work on Mac OS workstations for now.

So in my opinion, deployment wise, there should be no distinction between local / remote clusters.

Reply:
Removed from this page and will add to the QSG in the next commit

Reply:
I've updated the QSG but decided to skip the k2d mention altogether. Unless you say otherwise, based on my testing I don't think one needs it installed locally for the script to work - or am I wrong? (I do have it installed)